sky/utils/kubernetes/deploy_remote_cluster.sh

```diff
@@ -89,9 +95,19 @@ cleanup_server_node() {
     local NODE_IP=$1
     echo -e "${YELLOW}Cleaning up head node $NODE_IP...${NC}"
     run_remote "$NODE_IP" "
+        # Create temporary askpass script
+        ASKPASS_SCRIPT=\$(mktemp)
+        trap 'rm -f \$ASKPASS_SCRIPT' EXIT INT TERM ERR QUIT
+cat > \$ASKPASS_SCRIPT << EOF
+#!/bin/bash
+echo "$PASSWORD"
+EOF
+        chmod 700 \$ASKPASS_SCRIPT
+        # Use askpass
+        export SUDO_ASKPASS=\$ASKPASS_SCRIPT
         echo 'Uninstalling k3s...' &&
-        /usr/local/bin/k3s-uninstall.sh || true &&
-        sudo rm -rf /etc/rancher /var/lib/rancher /var/lib/kubelet /etc/kubernetes ~/.kube
+        sudo -A /usr/local/bin/k3s-uninstall.sh || true &&
```

Comment:
for discussion: would it be more standard if we prompt the user for a password by running the SSH commands with `ssh -T` instead of using askpass? The tradeoff is it makes it harder to write scripts using `sky local up`.  I'm ok using askpass, curious about your thoughts here.

Reply:
Based on the previous dicussion, the main issue with the prompt for password is that we need some handling for the HTTP request session, which may not worth the effort at the moment. We can get this pre-set approach in first. : )

Reply:
The client and server communicate with each other with HTTP for now, and it's not easy for the server to notify the client and receive input from it unless we introduce a mutual communication mechanism like websocket, however, I think it's not worth introducing websocket for this case only and there are also some side effects for websocket, e.g. persistent connection(and state) handling, scalability, etc. What do you think?

Reply:
Just for discussion: we may support prompting passwords on client-side only when `--password` flag is set to empty, just like most of other client tools, e.g. `mysql`

```
$ mysql -h 127.0.0.1 -p
Enter password: 
```

Reply:
Good point. After checking the code, the string option does not support setting to an empty value, so the validation will fail. If we need to support setting passwords from users, we need to add another option.